### PR TITLE
Add wrappers for readAttributes & subscribeToAttribute

### PR DESCRIPTION
**Pull request by mauimauer**
Allows to read and subscribe to arbitrary cluster attributes via ChipAttributePath. First step to fully solve issue #76 
Write and invoke need to be added and exposed via JNI on the Chip/Matter side

### CODE REVIEW

File: app/src/main/java/com/google/homesampleapp/chip/ChipClient.kt [hunk 1]
```diff
@@ -236,4 +236,65 @@ class ChipClient @Inject constructor(@ApplicationContext context: Context) {
     Timber.d("getDiscoveredDevice(${index})")
     return chipDeviceController.getDiscoveredDevice(index)
   }
+
```
Comment (pierredelisle) [inline app/src/main/java/com/google/homesampleapp/chip/ChipClient.kt:239]: Add comment for new logical section. E.g.
  // ---------------------------------------------------------------------------
  // Access clusters via numeric ids. Useful to access manufacturer specific clusters.
  // See https://github.com/google-home/sample-app-for-matter-android/issues/76.

File: app/src/main/java/com/google/homesampleapp/chip/ChipClient.kt [hunk 2]
```diff
@@ -236,4 +236,65 @@ class ChipClient @Inject constructor(@ApplicationContext context: Context) {
...
+        chipDeviceController.subscribeToAttributePath(
+            { continuation.resume(Unit) },
+            callback,
+            devicePtr,
+            listOf(attributePath),
+            minInterval,
+            maxInterval
+        )
+    }
+  }
```
Comment (pierredelisle) [inline app/src/main/java/com/google/homesampleapp/chip/ChipClient.kt:299]: Please run ktfmt to ensure standard formatting of all our source code.

**Review by pierredelisle — COMMENTED** [review]
Thanks for doing this Sebastian!

File: app/src/main/java/com/google/homesampleapp/chip/ChipClient.kt (hunk 2 above)
Comment (mauimauer) [reply app/src/main/java/com/google/homesampleapp/chip/ChipClient.kt:300]: Done

**Review by pierredelisle — APPROVED** [review]

**Pull request merged**